vm/internal/handler/import.go

```diff
@@ -76,6 +76,13 @@ func (h *Handler) ImportTarGzFile(ctx echo.Context) error {
 		return err
 	}
 
+	//fileName := filepath.Base(path)
+	rmCmd := "rm -rf /vackup-volume/..?* /vackup-volume/.[!.]* /vackup-volume/*"
```

Comment:
This command looks a bit over-complicated, why do we need this syntax ?

Reply:
This is necessary to remove hidden and not-hidden files and folders. I've added a comment with an explanation.